### DISCUSSION TITLE
GLPI-AGENT install with snap can't connect to server error certificate ssl how to enable no check ?

### DISCUSSION THREAD

**Discussion by SebaFR7474** [glpi-project/glpi-agent#807] [Q&A]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

Hi, i jusrt installed glpi-agent 1.11 on my glpi server dev to be added in glpi, and other linux.
he can't connect because or certificate failed, even if it's a good certificate.
How to encable the ssl check certificate, like the on on agent windows?

i used snap to install it.

And how to add glpi-agent at the start ?

thanks

**Comment by g-bougard** [comment]
Hello @SebaFR7474 

first as [described in documentation](https://glpi-agent.readthedocs.io/en/1.11/installation/index.html#snap), you can update configuration to, for example, enable debug2 level:
```
snap set glpi-agent debug=2
```

Fortunately, when debug=2 is enabled and `no-ssl-check` is also enabled (set to 1), you will be able to see in log a ssl fingerprint value you can directly use for `ssl-fingerprint` option. This is the best actually to authenticate via ssl a glpi server when its certificate is not trusted.

For the service, you should check the output of `sudo snap services`. In principle it should show you the service is installed. It may not be running if glpi-agent configuration doesn't permit it to continue, for example if none of server or local configuration is set.
